### PR TITLE
[LMS-84][Frontend] Selection Modal

### PR DESCRIPTION
**Pull request by mikorosales**
## Issue Link
[LMS-84](https://framgiaph.backlog.com/view/LMS-84)

## Defintion of Done
Selection Modal component should be created and is demonstrated with the use of a dynamic data table

## Steps to reproduce
1. docker-compose up
2. http://localhost:3000/demo/layouts/LearnersTab

## Affected Components / Functionalities / Page
n/a

## Test Cases
- [x] SelectionModal Reusable component should be created under shared > components
- [x] SelectionModal component can be expandable
- [x] SelectionModal component should work with data table selection
- [x] SelectionModal should only appear once there is atleast one selection made
- [x] SelectionModal should have a counter for each data being selected

## Notes
n/a

## Screenshots
058eb41a5b231dcfce654f054e0e4d73
image

### CODE REVIEW

**Comment by Marielleregner7** [comment]
Comment:
-  Modal should return Firstname and Lastname.
-  also please include delete/unselect button beside each content. (please refer to the image below) 

Thank you🙇 

mikomodal

Inspo:
image

**Comment by Marielleregner7** [comment]
Comment:
- I am seeing lint error. Please check image below 🙇
image

File: client/src/shared/components/SelectionModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}) => {
+  const [expanded, setExpanded] = useState(false);
+
+  const toggleExpand = (): void => {
+    setExpanded(!expanded);
+  };
+
+  const containerStyle = `fixed z-50 bottom-0 left-0 w-full max-h-96 ${
+    expanded ? 'h-1/3' : 'h-1/7'
+  } bg-white rounded-t-lg shadow-lg shadow-xl overflow-y-auto transition-all duration-500`;
```
Comment (ragg09-sunasterisk) [inline client/src/shared/components/SelectionModal/index.tsx:21]:
```suggestion
  } bg-white rounded-t-lg shadow-2xl shadow-black overflow-y-auto transition-all duration-500`;
```

**Review by ragg09-sunasterisk — APPROVED** [review]

**Review by Marielleregner7 — APPROVED** [review]
**APPROVED**
- [x] component can be expandable.
- [x] component works with data table selection
- [x] SelectionModal should only appear once there is atleast one selection made
- [ ] SelectionModal should have a counter for each data being selected
- [ ] No ESLint warnings or errors.
- [ ] No error found, when running npm run build.

Comments: 3
Test time: 60 mins
Iterations: 3

Proof:
selectionmodalmiko
image

**Pull request merged**